src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -439,9 +439,13 @@ object Strategy extends FgBioEnum[Strategy] {
     |   3. The assigned UMI tag
     |   4. Read Name
     |
+    |If the input is not template-coordinate sorted (i.e. `SO:unsorted GO:query SS:unsorted:template-coordinate`), then
+    |this tool will re-sort the input. The output will be written in template-coordinate order.
+    |
     |During grouping, reads are filtered out if a) all reads with the same queryname are unmapped, b) any primary
-    |read has mapping quality < `min-map-q` (default=1), or c) the primary mappings for R1 and R2 are on different
-    |chromosomes and `--allow-inter-contig` has been set to false.
+    |read has mapping quality < `min-map-q` (default=1), c) the primary mappings for R1 and R2 are on different
+    |chromosomes and `--allow-inter-contig` has been set to false., or d.) all non-primary reads are filtered,
+    |if `--includeSecondary` and\or `--includeSupplementary` are set to false (default=false)
```

Comment:
```suggestion
    |if `--include-secondary` and\or `--include-supplementary` are set to false (default=false)
```

Reply:
I don't grok 446-447 after multiple attempts at reading the usage (before looking at the code).  Perhaps you don't mean "non-primary"?  And if not, then I'm really confused!